docs/modules/ROOT/pages/how-tos/configuring/creating-secrets.adoc

```diff
@@ -93,6 +94,8 @@ Using the PAT authentication requires only the `password` key. The `username` sh
 
 This secret will be used by the build service to perform builds with Pipeline-as-Code.
 
+This Project Access Tokens are scoped to a project, so you cannot use them to access resources from other projects. For more information regarding Project Access Tokens, see the Gitlab https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html[documentation].
```

Comment:
What implications does this have if users want to onboard multiple projects? How can this be accomplished?

Reply:
That is already in the docs written just below these lines

Reply:
Ah, thanks.